### ISSUE TITLE
[SUGGESTION] Made you guys a AUR package with Linux suggestion

### ISSUE THREAD

**Issue by coreybruce** [nzp-team/nzportable#802]
Hey guys I just wanted to let you guys know I made you AUR package for you guys as I love this project and wanted to help out :smile: 
https://aur.archlinux.org/packages/nzp-bin

I have been downloading the linux binaries and re uploading them to my nzp repo https://gitlab.com/linuxbombay/nzp to easily version them and created a launch/setup script that is part of the package for a seamless user experience that will update the .config/NZP folder with the updated files from /user/share/games/NZP when the package is updated with it's active development and updates as quick as I can.

Also I have a suggestion to have the Linux binary create and use a nzp folder in .config for the game files including the nzp folder so there is a defined user folder location for the game files to go and for adding maps etc even if the game is part of a system package

Thanks for this amazing project and jeep up the good work!

**Comment by MotoLegacy** [comment]
thanks for this! super cool to see. I can investigate how we can navigate/approach loading assets from `.config/`, its tricky though since by nature of distribution the game assets being alongside the executable is pretty bog standard.

nitpick: any chance this can be renamed to `nzportable-bin`? :)

**Comment by coreybruce** [comment]
Awesome :) 

Yeah no problem if you think it should be named nzportable-bin instead I can make a new package and merge the current one with the correct name :) :+1:

**Comment by Peter0x44** [comment]
Another nit (I really, really doubt anyone cares):
```
source_i386=("https://gitlab.com/linuxbombay/nzp/nzp-packaging/-/archive/$scriptver/nzp-packaging-$scriptver.tar.bz2" "https://gitlab.com/linuxbombay/nzp/binaries/$pkgver/-/raw/main/nzportable-linux32.zip")
source_i686=("https://gitlab.com/linuxbombay/nzp/nzp-packaging/-/archive/$scriptver/nzp-packaging-$scriptver.tar.bz2" "https://gitlab.com/linuxbombay/nzp/binaries/$pkgver/-/raw/main/nzportable-linux32.zip")
```
Our "linux32" binaries are specifically i686. (disassembling them shows cmov and sse instructions).
https://archlinux32.org/architecture/
There is no mention of "i386" supported here, so I'm not sure what distro that would be for anyway.
There's absolutely no way such a CPU could run nzp anyway.

I think it's better you remove any mention to i386 since it is not serving a real purpose anyway.

**Comment by coreybruce** [comment]
Yeah no problem I can remove it, I doubt anyone would care if it was removed either lol

**Comment by Peter0x44** [comment]
I also noticed SDL2 isn't a runtime dependency. It should be.

**Comment by coreybruce** [comment]
Yeah I wasn't sure what the dependencies where haha but I'll add SDL2

**Comment by MotoLegacy** [comment]
everything else is statically linked, SDL2 is the only real dependency :) thank you for taking the time to address these!

**Comment by coreybruce** [comment]
No problem at all and I will try to update this as quick as I can when it comes to updates :smile: :+1:

**Comment by Peter0x44** [comment]
Why are "yad" and "rsync" dependencies?

**Comment by coreybruce** [comment]
I use yad for the graphical setup/launch script so the user can see whats going on before launch and I use rsync to compare and update the files between the /usr/share/games/NZP /home/$USER/.config/NZP if there is changes in the package folder

**Comment by Peter0x44** [comment]
I would have expected the package to place them in /usr/share, then you just rebuild the package when you want a newer version.

**Comment by coreybruce** [comment]
Oh what do you mean exactly sorry? the package files goes in /usr/share/games/NZP then the launch script will update the files in the .config folder and allow users to add custom maps to the user folder if they want to :)

**Comment by MotoLegacy** [comment]
Hi @coreybruce . We've made some changes to our game configuration files that need to be reflected on the package.

Namely, we now ship a "default.cfg" and a "nzportable.cfg" which can be safely modified and distributed. The game creates a "user_settings.cfg" on first-save that contains settings modified by the user which should be denylisted by your script in place of where you check against "fte.cfg" (which now no longer exists).

**Comment by coreybruce** [comment]
Hey @MotoLegacy Thanks for letting me know, I can look into that and make any changes if they are needed, the script pretty much copies all the files from the install location to the NZP user config folder to the appopiate location, is the "default.cfg" and "nzportable.cfg"  files in the root of the nzportable game folder or is it inside the nzp folder where the maps etc are?
